koans/1.intro.js

```diff
@@ -7,7 +7,7 @@ import {
 } from 'tape'
 
 import {
-  arrow,
+  arr,
```

Comment:
👍 
I think constructors/lifters should be ClassCased. It feel natural to work with the constructors that way, and it feels pretty natural to reference the types but capitalized names. Other than types/constructors, we only have functions and references, so there is nothing else to be capitalized. Capitalization signifies an extra level of formality that suits types very well, since types are indeed a more formalized construct than variable names. Seeing a capitalized name already prompts you to wonder what type class it belongs to and consequently what operations it supports; a plain function on the other hand you just expect to get some input and give you some output.

I'm also partial to full names, even if they mean typing more. The clarity that they bring is worth the extra effort. In particular in this case I would expect `arr` to mean array :D . Even if that's just me, I think it's worth removing the space for confusion.